src/main/scala/com/databricks/spark/xml/parsers/StaxXmlGenerator.scala

```diff
@@ -85,7 +86,8 @@ private[xml] object StaxXmlGenerator {
       case (StringType, v: String) => writer.writeCharacters(v)
       case (TimestampType, v: Timestamp) =>
         val formatter = options.timestampFormat.map(DateTimeFormatter.ofPattern).
-          getOrElse(DateTimeFormatter.ISO_INSTANT)
+          getOrElse(DateTimeFormatter.ISO_INSTANT).
```

Comment:
FWIW, I don't think this affects the port of spark-xml to Apache Spark. That code was modified to use Spark's more standard timestamp formatter code that always has a timezone available, it seems.